spring-cloud-gcp-docs/src/main/asciidoc/spring-integration-pubsub.adoc

```diff
@@ -144,3 +144,12 @@ public void handleMessage(Message<?> msg) throws MessagingException {
 ----
 
 It is also possible to set an SpEL expression for the topic with the `setTopicExpression()` or `setTopicExpressionString()` methods.
+
+==== Header mapping
```

Comment:
Would be great to mention somehow in this paragraph our `PubSubHeaderMapper` implementation.
Otherwise it isn't clear how end-user can configure a desired set of headers.
Some code snippet would be great addition too, as well as a sample project modification on the matter.